src/configupdater/option.py

```diff
@@ -71,6 +80,7 @@ def _join_multiline_value(self):
             self._multiline_value_joined = True
 
     def __str__(self) -> str:
+        print(self.lines)
```

Comment:
This looks like a debug statement left.

```suggestion
```

Reply:
Thanks! Forgot that, it's a bit embarrassing :-)

Reply:
😝  (I always leave debugging statements behind)